client/routes/execution/history.vue

```diff
@@ -326,55 +423,74 @@ section.history
     border brdr
     overflow auto
 
-  table
-    thead
+  .table
+    .vue-recycle-scroller__slot,
+    .vue-recycle-scroller__item-view,
+    .scroller-item
+      display flex
+      width 100%
+    .col-id
+      min-width 50px
+    .col-summary
+      flex 1
+    .col-time
+      min-width 150px
+    .col-type
+      min-width 212px
+    .thead
       background-color uber-white-10
       box-shadow 2px 2px 2px rgba(0,0,0,0.2)
-      &.floating
-        position absolute
-        display flex
-        top 0
-        left 0
-        z-index 2
-        th
-          display inline-block
-          & > .v-select.eventType
-            margin-left 10px
-            display: inline-block
-            width: 150px
-        & + .spacer
-          width 100%
-          height 38px
-    td, th
-      &:first-child
-        min-width 60px
-      &:nth-child(2)
-        min-width 150px
-    th a:not([href])
+      position absolute
+      display flex
+      top 0
+      left 0
+      z-index 2
+      width calc(100% - 10px)
+      .th
+        color: rgb(0, 0, 0);
+        display inline-block
+        font-weight: 500;
+        text-transform: uppercase;
```

Comment:
You have semicolons here but not above or below. Is this correct?

Reply:
Can update, unfortunately the whole codebase in cadence web is missing semicolins. I will update this bit and maybe open a separate PR to update all of them in one hit. Thanks!